src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -1,13 +1,27 @@
 import React, { useContext } from 'react';
 import { node } from 'prop-types';
 import GlobalStyles from '@bbc/psammead-styles/global-styles';
+import styled from 'styled-components';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
 import HeaderContainer from '../containers/Header';
 import FooterContainer from '../containers/Footer';
 import ManifestContainer from '../containers/Manifest';
 import ServiceWorkerContainer from '../containers/ServiceWorker';
 import { ServiceContext } from '../contexts/ServiceContext';
 import WebVitals from '#app/containers/WebVitals';
 
+const Wrapper = styled.div`
+  min-height: 100%;
```

Comment:
We use `min-height: 100vh;` on `latest` right now.

Reply:
min-height: 100% in this case will be 100vh so let's just use % since it has more browser support

Reply:
You're correct @simonsinclair we need to use vh https://github.com/bbc/simorgh/pull/8266